probe/docker/reporter.go

```diff
@@ -191,30 +192,35 @@ func (r *Reporter) containerTopology(localAddrs []net.IP) report.Topology {
 				Add(ContainerIPsWithScopes, report.MakeStringSet(hostIPsWithScopes...))
 		}
 
-		var networkInfo func(prefix string) report.Sets
-		networkInfo = func(prefix string) report.Sets {
+		var networkInfo func(prefix string) (report.Sets, bool)
+		networkInfo = func(prefix string) (ips report.Sets, isHostMapped bool) {
```

Comment:
I'm not normally a stickler for names, but can we call this something else?  like `inHostNamespace`?

Also pls put a comment about how we can't use the networkmode to detect this, due to `--net=container:foo`